h2o-py/h2o/grid/grid_search.py

```diff
@@ -175,6 +180,7 @@ def train(self, x=None, y=None, training_frame=None, offset_column=None, fold_co
         # dictionaries have special handling in grid search, avoid the implicit conversion
         parms["search_criteria"] = None if self.search_criteria is None else str(self.search_criteria)
         parms["export_checkpoints_dir"] = self.export_checkpoints_dir
+        parms["parallelism_level"] = self._parallelism
```

Comment:
I prefer `parallelism` everywhere: wondering why you have `parallelism_level` or `parallelismLevel` on backend/RestAPI